service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EpisodeOfCareMapper.java

```diff
@@ -49,7 +49,7 @@
 
             if (assignedEntity.isSetRepresentedOrganization()) {
                 POCDMT000002UK01Organization representedOrganization = assignedEntity
-                    .getRepresentedOrganization();
+                        .getRepresentedOrganization();
```

Comment:
please remove additional spacing here

Reply:
This is the code formatter, not added intentionally